src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -294,15 +295,24 @@ internal string? DefaultUserId
 
     internal List<string> SdkIntegrationNames { get; set; } = new();
 
+    internal ISentryUnityInfo UnityInfo { get; private set; }
+    internal Action<SentryUnityOptions> PlatformConfiguration { get; private set; }
+
     public SentryUnityOptions() : this(false, ApplicationAdapter.Instance) { }
 
     internal SentryUnityOptions(bool isBuilding, IApplication application, ISentryUnityInfo? unityInfo = null) :
         this(SentryMonoBehaviour.Instance, application, isBuilding, unityInfo)
     { }
 
     // For testing
-    internal SentryUnityOptions(SentryMonoBehaviour behaviour, IApplication application, bool isBuilding, ISentryUnityInfo? unityInfo = null)
+    private SentryUnityOptions(SentryMonoBehaviour behaviour, IApplication application, bool isBuilding, ISentryUnityInfo? unityInfo)
     {
+        // This should never happen. The PlatformServices are set through the RuntimeLoad attribute in 'SentryInitialization.cs'
```

Comment:
I debug a few "this should never happen" things in my life to be more humble LOL
how about:

```suggestion
        // This shouldn't happen. The PlatformServices are set through the RuntimeLoad attribute in 'SentryInitialization.cs'
```